corgi/core/models.py

```diff
@@ -395,7 +406,7 @@ def builds(self) -> QuerySet:
         if product_refs:
             return (
                 ProductComponentRelation.objects.filter(product_ref__in=product_refs)
-                .values_list("build_id", flat=True)
+                .values_list("build_id", "build_type")
```

Comment:
On second though, if we did that it would mean you could not have a SoftwareBuild linked to a Product without also having a component. I prefer it the way it is, without the dependency on a Component.